OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -36,6 +36,14 @@ public class CashTricklerInfo : PausableConditionalTraitInfo
 		[Desc("Use resource storage for cash granted.")]
 		public readonly bool UseResourceStorage = false;
 
+		void IRulesetLoaded<ActorInfo>.RulesetLoaded(Ruleset rules, ActorInfo info)
+		{
+			var tricklers = info.TraitInfos<CashTricklerInfo>();
```

Comment:
I may be missing something, but wouldn't this be sufficient?

```csharp
		void IRulesetLoaded<ActorInfo>.RulesetLoaded(Ruleset rules, ActorInfo info)
		{
			if (ShowTicks && !info.HasTraitInfo<IOccupySpaceInfo>())
				throw new YamlException("CashTrickler is defined with ShowTicks 'true' and the actor occupies no space.".F(info.Name));
		}
```

Reply:
I think you are right. This seems to be looking up other cash tricklers as well.